portfolio/src/main/webapp/about.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
```

Comment:
Put this line right below the DOCTYPE

Reply:
Sorry, I'm not sure I understand. Which line?

Reply:
<!DOCTYPE html> 
<meta charset="UTF-8">
Did the review comment not highlight which line this comment referred to? This applies to all the other comments below

Reply:
Also, I think the "utf-8" should be lowercased: https://google.github.io/styleguide/htmlcssguide.html#Encoding

Reply:
It looks like 3-4 lines have been highlighted. I'm a little confused about what I should do.

Reply:
This was the formatting I referred to 
<!DOCTYPE html>
<meta charset="utf-8">
Thanks Kelly for the capitalization pointer